src/st/compatibility/class-wpml-page-builders-translatable-nodes.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+abstract class WPML_Page_Builders_Translatable_Nodes {
```

Comment:
@adrianosferreira Should this have `implements IWPML_Page_Builders_Translatable_Nodes`?

Reply:
@brucepearson I don't think so. Child classes like `WPML_VC_Website_Builder_Translatable_Nodes` will implement `IWPML_Page_Builders_Translatable_Nodes` so they know which methods must be declared.

Reply:
@adrianosferreira Are you sure because this looks like it class does implement the interface.

Reply:
@brucepearson done.